checkups/echo/build-image

```diff
@@ -3,6 +3,13 @@
 set -x
```

Comment:
This change does not belong to this PR.

Reply:
I added this commit to this PR because its the closest thing to an end-to-end test.
Using it enables you to verify that running a checkup though the framework actually works.

I will move it to another PR.